src/templates/userupdate.html

```diff
@@ -1,40 +1,130 @@
-{% for err in errors %}
-<div class="alert alert-danger" role="alert">
-{{ err }}
-</div>
+{% extends "layouts/base.html" %}
+{% block title %}User information edit{% end %}
+{% block content %}
+<style>
```

Comment:
Where is that? I've looked into scq/src/templates/layouts/standard/head which has a file called "javascripts.html"